addons/dark-www/addon.json

```diff
@@ -2333,6 +2355,44 @@
         "darkForumCode": true
       }
     },
+    {
+      "name": "Scratch 2.0 (2013)",
+      "id": "scratchr2-blue",
+      "description": "The colors previously used on Scratch 2.0-styled pages",
+      "values": {
+        "page": "#fcfcfc",
+        "navbar": "#0f8bc0",
+        "box": "#ffffff",
+        "gray": "#f7f7f7",
+        "blue": "#e6f3fa",
+        "input": "#ffffff",
+        "button": "#18abeb",
+        "link": "#1aa0d8",
+        "footer": "#ececec",
+        "border": "#0000001a",
+        "darkBanners": "off",
+        "darkForumCode": false
+      }
+    },
+    {
+      "name": "Scratch 2.0 (2015)",
+      "id": "scratch-oldWwwBlue",
+      "description": "The colors of the home page between 2015 and 2018",
+      "values": {
+        "page": "#fcfcfc",
+        "navbar": "#25aff4",
+        "box": "#ffffff",
+        "gray": "#f2f2f2",
+        "blue": "#e6f3fa",
+        "input": "#fafafa",
+        "button": "#25aff4",
+        "link": "#25aff4",
+        "footer": "#f2f2f2",
+        "border": "#0000001a",
+        "darkBanners": "off",
+        "darkForumCode": false
+      }
+    },
```

Comment:
I feel like some people may not remember this one. I remember that change, but it's pretty subtle and by now, I don't think anyone would prefer it over the deeper blue (the one before mid-2023). I would skip this preset as to not add too many, but that's just my opinion.

Reply:
I agree about the 2013 preset being more important. Not sure if that means we shouldn't add this one, though.